doc/command-line.md

```diff
@@ -109,16 +145,199 @@ Options:
+high [Jaccard similarity](https://en.wikipedia.org/wiki/Jaccard_index)
+or containment; the default is to use Jaccard similarity, unless
+`--containment` is specified.  `-o/--output` will create a CSV file
+containing the matches.
+
+`search` will load all of provided signatures into memory, which can
+be slow and somewhat memory intensive for large collections.  You can
+use `sourmash index` to create a Sequence Bloom Tree (SBT) that can
+be quickly searched on disk; this is [the same format in which we provide
+GenBank and other databases](databases.html).
+
+Usage:
+```
+sourmash search query.sig [ list of signatures or SBTs ]
+```
+
+Example output:
+
+```
+49 matches; showing first 20:
+similarity   match
+----------   -----
+ 75.4%       NZ_JMGW01000001.1 Escherichia coli 1-176-05_S4_C2 e117605...
+ 72.2%       NZ_GG774190.1 Escherichia coli MS 196-1 Scfld2538, whole ...
+ 71.4%       NZ_JMGU01000001.1 Escherichia coli 2-011-08_S3_C2 e201108...
+ 70.1%       NZ_JHRU01000001.1 Escherichia coli strain 100854 100854_1...
+ 69.0%       NZ_JH659569.1 Escherichia coli M919 supercont2.1, whole g...
+...    
+```
+
+### `sourmash gather`
+
+The `gather` subcommand finds all non-overlapping matches to the
+query.  This is specifically meant for metagenome and genome bin
+analysis.  (See [Classifying Signatures](classifying-signatures.html)
+for more information on the different approaches that can be used
+here.)
+
+If the input signature was computed with `--track-abundance`, output
+will be abundance weighted (unless `--ignore-abundances` is
+specified).  `-o/--output` will create a CSV file containing the
+matches.
+
+`gather`, like `search`, will load all of provided signatures into
+memory.  You can use `sourmash index` to create a Sequence Bloom Tree
+(SBT) that can be quickly searched on disk; this is
+[the same format in which we provide GenBank and other databases](databases.html).
+
+Usage:
+```
+sourmash gather query.sig [ list of signatures or SBTs ]
+```
+
+Example output:
+```
+overlap     p_query p_match 
+---------   ------- --------
+1.4 Mbp      11.0%   58.0%      JANA01000001.1 Fusobacterium sp. OBRC...
+1.0 Mbp       7.7%   25.9%      CP001957.1 Haloferax volcanii DS2 pla...
+0.9 Mbp       7.4%   11.8%      BA000019.2 Nostoc sp. PCC 7120 DNA, c...
+0.7 Mbp       5.9%   23.0%      FOVK01000036.1 Proteiniclasticum rumi...
+0.7 Mbp       5.3%   17.6%      AE017285.1 Desulfovibrio vulgaris sub...
+```
+
+Note:
+
+Use `sourmash gather` to classify a metagenome against a collection of
+genomes with no (or incomplete) taxonomic information.  Use `sourmash
+lca summarize` and `sourmash lca gather` to classify a metagenome
+using a collection of genomes with taxonomic information.
+
+## `sourmash lca` subcommands
+
+These commands use LCA databases (created with `index`, below, or
+prepared databases such as
+[genbank-k31.lca.json.gz, from the LCA tutorial](tutorials-lca.html).
+
```

Comment:
I thought `index` was to create the SBT for gather?

Reply:
`lca index`. Fixed!